src/shared/bond-util.h

```diff
@@ -0,0 +1,106 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+#include <netinet/in.h>
+#include <linux/if_bonding.h>
+
+#include "in-addr-util.h"
```

Comment:
Why is this necessary?

Reply:
yes It's not compiling.

Reply:
all you need is to include "macro.h", so that you get `_pure_` and `_const_`. If you include that instead of in-addr-util.h all should be good.